example/src/main/resources/logback.xml

```diff
@@ -31,8 +31,9 @@
     <logger name="akka" level="WARN"/>
     <logger name="slick" level="INFO"/>
     <logger name="io.scalac" level="INFO"/>
+    <logger name="io.scalac.mesmer.extension.AkkaMonitoring" level="DEBUG"/>
 
-    <logger name="akka.persistence" level="TRACE" additivity="false">
+    <logger name="io.scalac.mesmer.utils" level="DEBUG" additivity="false">
```

Comment:
Yeah, this might not be wanted, but I don't think we need `akka.persistence` set to trace, and this one redirects logs from a little agent that logs all loaded classes and classloader that  loaded it, which is pretty useful to me 🙃

Reply:
There're only `agent.utils` and `core.util` packages. Does this logback change work?

Reply:
Yeah, this is for something outside of the current repo - https://github.com/worekleszczy/monitor-agent

Reply:
isn't java's `-verbose:class` option enough?

Reply:
I agree that we should not trace akka persistence, so I suggest removing both:

```suggestion
```

Reply:
I also checked if java -verbose:class is enough but it does not seem to show you the classloader

Reply:
I agree that we can remove this here . I don't think `-verbose:class` is enough for my use case - I tried using this before I created another agent and had a hard time redirecting it to another file. This should be possible with new java9 syntax for logs, but I didn't succeed in doing it.